src/Animations/babylon.animatable.ts

```diff
@@ -99,44 +99,51 @@
         }
 
         public stop(animationName?: string): void {
-            var idx = this._scene._activeAnimatables.indexOf(this);
-
-            if (idx > -1) {
-
-                if(animationName){
+            
+            if (animationName) {
+                
+                var idx = this._scene._activeAnimatables.indexOf(this);
+                
+                if (idx) {

```

Comment:
You might be correct.

This fix was a quick fix to get the working code from 2.4 back into 2.6.

The animationName option that was added to 2.5 caused the bug.